### PR TITLE
TIP-888: Prepare standard for 3.0

### PR DESCRIPTION
**Pull request by jjanvier**
- 1 missing information added in the UPGRADE.md
- missing injections added (caused by a refactor)

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -165,7 +165,13 @@ TODO: change the link!!
     * the route `oro_user_security_check` has been replaced by `pim_user_security_check`
     * the route `oro_user_security_login` has been replaced by `pim_user_security_login`
     * the route `oro_user_security_logout` has been replaced by `pim_user_security_logout`
-
+    * the Elasticsearch configurations files is now:
```
Comment (anaelChardan) [inline UPGRADE.md]: configurations file*

File: UPGRADE.md [hunk 2]
```diff
@@ -165,7 +165,13 @@ TODO: change the link!!
     * the route `oro_user_security_check` has been replaced by `pim_user_security_check`
     * the route `oro_user_security_login` has been replaced by `pim_user_security_login`
     * the route `oro_user_security_logout` has been replaced by `pim_user_security_logout`
-
+    * the Elasticsearch configurations files is now: 
+    TODO
+    ```yaml
+    elasticsearch_index_configuration_files
```
Comment (anaelChardan) [inline UPGRADE.md]: it misses `:`

**Comment by jjanvier** [comment]
CI OK (and launched, fixed a CS, and rebased)

File: UPGRADE.md [hunk 3]
```diff
@@ -165,7 +165,13 @@ TODO: change the link!!
     * the route `oro_user_security_check` has been replaced by `pim_user_security_check`
     * the route `oro_user_security_login` has been replaced by `pim_user_security_login`
     * the route `oro_user_security_logout` has been replaced by `pim_user_security_logout`
-
+    * the Elasticsearch configuration files is now: 
+    TODO
+    ```yaml
+    elasticsearch_index_configuration_files:
+        - '%pim_ce_dev_src_folder_location%/src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml'
```
Comment (arnolanglade) [inline UPGRADE.md]: Old path

**Pull request merged**